hw8.py

```diff
@@ -0,0 +1,69 @@
+import os
+import random
+
+os.system("clear")
+
+
+def BullsCows(number, attempt):
+    bulls = 0
+    cows = 0
+
+    for i in range(4):
+        if number[i] == attempt[i]:
+            bulls += 1
+
+    for j in attempt:
+        if j in number:
+            cows += 1
```

Comment:
You can simplify your code by removing unnecessarily loops

```suggestion
    for i in range(len(attempt)):
        if number[i] == attempt[i]:
            bulls += 1
        elif number[i] in attempt:
            cows += 1
```